src/hevm/src/EVM.hs

```diff
@@ -2085,6 +2142,17 @@ costOfCall (FeeSchedule {..}) recipientExists xValue availableGas xGas =
       if availableGas >= c_extra
       then min xGas (allButOne64th (availableGas - c_extra))
       else xGas
+    c_foo = trace ("\n" <> " c_new:     " <> show c_new  <> "\n"
```

Comment:
is this supposed to be here? Or is this debugging material sneaking into production

Reply:
oops yes, will remove